spec/contact_spec.rb

```diff
@@ -1,14 +1,16 @@
 ENV['RACK_ENV'] = 'test'
-ENV['DESKPRO_API_KEY'] = FAKE_DESKPRO_API_KEY
```

Comment:
should we remove these? you seem to test both cases when zendesk is enabled or disabled

Reply:
We should keep it. Anyway, using these `ENV['whatever']` here is wrong, because it actually gets executed for all the test files when loading all the modules. 

I am going to move this to only one place.

Reply:
I moved them.